### PR TITLE
Update for recent crafter capabilities changes.

### PR DESCRIPTION
**Pull request by Mekle001**

### CODE REVIEW

File: source/buildings/sawmill.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ layout: default
 ### Note: The Sawmill cannot be built until you have a level 3 [Lumberjack's Hut](../../source/buildings/lumberjack) (or three level 1 Lumberjack's Huts, or another equivalent) and have finished the research in the [University](../../source/buildings/university).
 <br>
 
-The Sawmill is where the Miller will craft any items made of at least 75% wood (no bows or redstone items). For them to do this, you must teach the Sawmill the recipes and the Miller must receive a request for an item from another worker.
+The Sawmill is where the Miller will craft any items made of at least 75% wood (no bows or redstone items). The sawmill will also craft a few specialty items including cactus planks, shingles and timber frames. For them to do this, you must teach the Sawmill the recipes and the Miller must receive a request for an item from another worker.
```
Comment (ravenbuilder934) [inline source/buildings/sawmill.md:25]: Aren't cactus planks, shingles, and timber frames at least 75% wood?

File: source/buildings/stonemason.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ layout: default
 ### Note: The Stonemason's Hut cannot be built until you have a level 1 [Blacksmith's Hut](../../source/buildings/blacksmith) and have finished the research in the [University](../../source/buildings/university).
 <br>
 
-The Stonemason's Hut is where the Stonemason will craft 3x3 recipes made entirely out of cobblestone, stone, andesite, diorite, granite, sandstones, and/or ores (no quartz, purpur, endstone, or ingots). You will need to teach the Stonemason's Hut the recipes you want the Stonemason to craft. The Stonemason will only work when they receive a request and they have the needed materials.
+The Stonemason's Hut is where the Stonemason will craft 3x3 recipes made entirely out of cobblestone, stone, andesite, diorite, granite, sandstones. The stonemason has recently learned out to create purpur, work with quartz blocks made by the mechanic, nether bricks made by the stonesmelter, and even some tricks with prismarine. You will need to teach the Stonemason's Hut the recipes you want the Stonemason to craft. The Stonemason will only work when they receive a request and they have the needed materials.
```
Comment (ravenbuilder934) [inline source/buildings/stonemason.md:25]: Please remove this, as I already added it to my PR with more detail.

File: source/buildings/stonesmeltery.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ layout: default
 ### Note: The Stone Smeltery cannot be built until you have a level 3 [Smeltery](../../source/buildings/smeltery) (or three level 1 Smelteries, or another equivalent) and have finished the research in the [University](../../source/buildings/university).
 <br>
 
-The Stone Smeltery is where the Stone Smelter will smelt cobblestone into stone, stone bricks into cracked stone bricks, clay balls into bricks, and clay blocks into terracotta. The Stone Smelter will only do this if they receive a request from another worker and they have the needed materials/fuel.
+The Stone Smeltery is where the Stone Smelter will smelt cobblestone into stone, stone bricks into cracked stone bricks, clay balls into bricks, and clay blocks into terracotta. The stone smelter can also process all the smooth stone smelting, and recently learned how to pop chorus fruit for the stonemason. The Stone Smelter will only do this if they receive a request from another worker and they have the needed materials/fuel.
```
Comment (ravenbuilder934) [inline source/buildings/stonesmeltery.md:25]: Please remove this, as I already added it to my PR with more detail.

**Comment by ravenbuilder934** [comment]
Also don't forget to change the Baker worker page.

File: source/buildings/bakery.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ When accessing the Bakery block by right-clicking on it, you will see a GUI with
 
 On page two of the GUI, you will see a list of recipes that the Baker is able to make. Just click to change the ones you want the Baker to bake to on and they will begin making those items as long as they receive the required ingredients. (The black box at the top is to search for recipes.) The bakery already knows the recipes for bread, cakes, cookies, and pumpkin pies without you having to teach them. 
 
-You can still teach the baker those recipes, to enable them to be made on demand, such as when the builder needs Cake for a decoration. 
+You can still teach the Baker those recipes to enable them to be made on demand, such as when the [Builder](../../source/workers/builder) needs cake as a decoration.
```
Comment (ravenbuilder934) [inline source/buildings/bakery.md:51]:
```suggestion
You can still teach the Bakery those recipes to enable them to be made on demand, like if the [Builder](../../source/workers/builder) needs cake as a decoration. 
```

File: source/buildings/sawmill.md (hunk 1 above)
Comment (Mekle001) [reply source/buildings/sawmill.md:25]: Not necessarily. Timber frames are 33% wood in most cases, with shingles being ~50%. Cactus doesn't qualify as wood by default.

Comment (ravenbuilder934) [reply source/buildings/sawmill.md:25]: Then

```suggestion
The Sawmill is where the Miller will craft any items made of at least 75% wood (no bows or redstone items). The Miller will also craft a few other items, including cactus planks, [shingles](../../source/items/shingles), and [timber frames](../../source/items/timberframes). For them to do this, you must teach the Sawmill the recipes and the Miller must receive a request for an item from another worker.
```

Comment (ravenbuilder934) [reply source/buildings/sawmill.md:25]: Also change this on the Miller worker page.

File: source/buildings/dyer.md [hunk 5]
```diff
@@ -22,7 +22,10 @@ layout: default
 ### Note: The Dyer cannot be built until you have a level 3 [Flower Shop](../../source/buildings/flowershop) (or three level 1 Flower Shops, or another equivalent) and have finished the research in the [University](../../source/buildings/university).
 <br>
 
-The Dyer's Hut is where the Dyer will craft dyes and dye other items (except for concrete). The Dyer will only do this if they receive a request from another worker and have the necessary materials. The Dyer's Hut automatically knows the recipe to make green dye from cacti, but you must teach it the recipes for the other dyes and the dyed items.
+The Dyer's Hut is where the Dyer will craft dyes and dye other items (except for concrete). The Dyer will only do this if they receive a request from another worker and have the necessary materials. The Dyer's Hut automatically knows the recipe to make green dye from cacti and red sand from sand and cobblestone, but you must teach it the recipes for the other dyes and almost all dyed items except for colored concrete powder.
+
+Additionally, the dyer can make red nether brick.
```
[the remark this answers is not in the document]
Comment (ravenbuilder934) [reply source/buildings/dyer.md:27]: Why did you resolve this? It isn't removed.

**Comment by ravenbuilder934** [comment]
Also change the Miller worker page.

File: source/buildings/dyer.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Mekle001) [reply source/buildings/dyer.md:27]: Clicked the wrong one, missed this one.

**Pull request merged**